### PR TITLE
test pr

### PR DESCRIPTION
**Pull request by condret**
- [ ] Mark this if you consider it ready to merge
- [ ] I've added tests (optional)
- [ ] I wrote some lines in the [book](https://github.com/radareorg/radare2book) (optional)

**Description**

### CODE REVIEW

File: libr/asm/p/asm_arm_cs.c [hunk 1]
```diff
@@ -279,38 +59,16 @@ static int assemble(RAsm *a, RAsmOp *op, const char *buf) {
...
-	return 0;
-}
-
 RAsmPlugin r_asm_plugin_arm_cs = {
 	.name = "arm",
 	.desc = "Capstone "CAPSTONE_VERSION_STRING" ARM disassembler",
 	.cpus = ",v8,cortex",
 	.features = "v8",
-	.license = "BSD",
+	.license = "LGPL",
```
Comment (trufae) [inline libr/asm/p/asm_arm_cs.c:67]: capstone is bsd
```suggestion
	.license = "BSD",
```

**Review by trufae — APPROVED** [review]
LGTM

File: libr/anal/p/anal_arm_cs.c [hunk 2]
```diff
@@ -4439,98 +4440,190 @@ static void op_fillval(RAnal *anal, RAnalOp *op, csh handle, cs_insn *insn, int
 	}
 }
 
+static R_TH_LOCAL csh cs_handle = 0;
```
Comment (trufae) [inline libr/anal/p/anal_arm_cs.c:4443]: There's a PR from @iotaaxel moving those variables into RAnal. will be merged after 5.7.8, so we can make capstone plugins a bit more threadsafe

Comment (condret) [reply libr/anal/p/anal_arm_cs.c:4443]: sure, but I want to merge this tonight

**Pull request merged**